fuse.py

```diff
@@ -644,7 +644,8 @@ def setxattr(self, path, name, value, size, options, *args):
 
     def getxattr(self, path, name, value, size, *args):
         ret = self.operations('getxattr', path.decode(self.encoding),
-                                          name.decode(self.encoding), *args)
+                                          name.decode(self.encoding), *args) \
+                  .encode(self.encoding)
```

Comment:
Currently the encoding only happens automatically on path and attribute names. Your operations should return bytes for the attribute values rather than relying on fusepy to encode the data for you. The whole encoding issue seems to have caused problems of its own (#2), but I drew the line at the data that was being stored in the files and the attribute values. In Python3 the distinction is more obvious and forces you to be more diligent about encoding data where Python2 the issue is swept under the rug and most people don't realize the importance of encodings.